src/dev-core/src/app/button/button.demo.scss

```diff
@@ -0,0 +1,4 @@
+.inverse-btn-container {
+  padding: 0.5rem 0.5rem;
+  background-color: hsl(0, 0%, 19%);
```

Comment:
This is `$clr-color-neutral-900` or `var(--clr-color-neutral-900, #{$clr-color-neutral-900})`;

Reply:
This is same as above comment. Do you feel strongly that I should import clarity global variables and color variables in the demo page's scss file?

Reply:
I’m on the fence about it. I leave it to your best judgment.

Reply:
It's not a strong stance for me.. so I'm happy to import the variables and update the demo page. Will do that.